### ISSUE TITLE
[Rendering][Authoring][Designer] aria-label attribute is not available for elements; it is necessary for screen reader support

### ISSUE THREAD

**Issue by xyzzer-msft** [microsoft/AdaptiveCards#8394]
### Target Application

Widgets Board

### Application Operating System

Windows

### Schema Version

Latest

### Problem Description

When designing any UI that isn't a straightforward document - you need different affordances for using the UI for users who can see and for users who can't. When you can see the UI layout, hover over elements and see tooltips - you can infer much more than if you use it with a screen reader. Aria-label is the web version of the most basic screen reader affordance and it is missing in all or most cases from adaptive cards at least based on what I can infer from the designer. The aria-label may be the narrated description of an icon seen on a button that makes sense to hear. You can provide a tooltip for some elements like buttons, but a tooltip should provide more information than what the button itself already provides and should not repeat what's already seen when looking at a button, while an aria-label might serve that purpose. Also, it may be useful for more than just a button. Labels (TextBlock), Input.Text, Input.Number, etc and any other elements should support both tooltips and aria-label.

### Expected Outcome

Aria-label or AutomationProperties.Name-like label for screen readers available on any element.

### Actual Outcome

missing

### Card JSON

```JSON
-
```

**Comment by xyzzer-msft** [comment]
Possibly related to https://github.com/microsoft/AdaptiveCards/issues/5383

**Comment by jwoo-msft** [comment]
FYI, @JeanRoca please feel free to lower the priority.

**Comment by jwoo-msft** [comment]
@xyzzer-msft, can you add json for this use case?

**Comment by xyzzer-msft** [comment]
It's a bit difficult to add a json for something that is missing, especially for an internal project. Please reach out to me via Teams and I can share a Figma.
